### DISCUSSION TITLE
Nuxt 2 and onesignal

### DISCUSSION THREAD

**Discussion by pmudassir** [nuxt/nuxt#30235] [Questions]
Has anyone of you worked with nuxt 2 and onsesignal, if yes please share code example or anything that can help me get this done. Thanks in advance

**Comment by e112121** [comment]
https://github.com/antfu/vitesse-nuxt-bridge

**Reply by pmudassir** [reply]
whats this?

**Reply by e112121** [reply]
Please take a look at the Nuxt 2 documentation and the description of this project first. Don’t ask questions without thinking, or no one will answer your questions.
